libs/core/src/lib/token/tokenizer.component.ts

```diff
@@ -119,12 +119,29 @@ export class TokenizerComponent implements AfterViewChecked, AfterViewInit, Afte
     /** @hidden */
     hiddenCozyTokenCount = 0;
 
+    /** @hidden */
+    shiftKeyPressed = false;
+
+    /** @hidden */
+    ctrlKeyPressed = false;
+
+    /** @hidden */
+    private firstElement: number;
+
+    /** @hidden */
+    private lastElement: number;
+
+    /** @hidden */
+    private ctrlPrevious: boolean;
+
+    /** @hidden */
+    private directionRight: boolean;
```

Comment:
Could you make any comments, or change naming, I got no clue what's going on with those variables